spec/requests/admin/admin_portals_spec.rb

```diff
@@ -3,7 +3,11 @@
 RSpec.describe "/admin", type: :request do
   let(:super_admin) { create(:user, :super_admin) }
 
-  before { sign_in super_admin }
+  before do
+    sign_in super_admin
+    allow(FeatureFlag).to receive(:enabled?).with(:profile_admin).and_return(false)
```

Comment:
I have to stub out both the profile admin and the DUS Feature flag because it errors with 
```
        Please stub a default value first if message might be received with other args as well. 
```
Not sure if there's a better way to do this instead of stubbing both out all the time.

Reply:
I was thinking of adding both to the rail helper [here](https://github.com/forem/forem/blob/master/spec/rails_helper.rb#L183) but not sure if it's the best way. Any thoughts?

Reply:
I wonder if we can't just enable the flag and disable it in the `after` block, to clean up

Reply:
Ah I think I figured it out -  I just need to have this in the before 
```allow(FeatureFlag).to receive(:enabled?).and_call_original``` 
instead of specifically setting the unneeded feature flag.

This will configure a default response for most arguments, and then override
it for specific arguments using the with.

which is what I think my initial error message was hinting at:
```
Please stub a default value first if message might be received with other args as well. 
```